### ISSUE TITLE
crash connecting to remote graph

### ISSUE THREAD

**Issue by rdp** [cplussharp/graph-studio-next#279]
Hello.
As a note if I install "direct show spy" dll's then launch SichboPVR, it adds two graphs to the ROT while running.  Opening up one of them using graph studio next -> crash (graphedit can load it OK).  Let me know what further information I can provide or where it might be located.  Cheers!

**Comment by roman380** [comment]
Do I get this right, that with spy you have two entries in both GraphEdit, and GraphStudioNext and GE opens both graphs, GSN opens neither of the two (crashes). Then both GE and GSN are 32-bit apps?

**Comment by rdp** [comment]
yes two entries show up "everywhere."  GE can open both (32 bit and 64 bit
GE both can open both remote graphs).  With GSN 32 or 64 bit, it opens one
OK, and "dies" opening the other.  If that makes sense.

**Comment by roman380** [comment]
My guess is that the problem is more likely to be with spy rather than with GSN, it might be bug or some misfit. I installed the app to possibly see it live, but I don't have hardware to reproduce the entire scenario.

**Comment by rdp** [comment]
Interesting.  If I try and save it from graphedit, either graph they both
say "An error occurred while attempting to open the file. No such interface
supported (Return code 0x80004002)." and it proceeds to not create a .grf
file. Odd.

**Comment by mikecopperwhite** [comment]
There is a known crash connecting to remote graphs in issue #333. Not clear if this is the same issue without more details.
